mkt/websites/models.py

```diff
@@ -17,7 +20,7 @@ class Website(ModelBase):
     title = TranslatedField()
     short_title = TranslatedField()
     description = TranslatedField()
-    keywords = models.ManyToManyField(Tag)
+    _keywords = models.ManyToManyField(Tag, db_column='keywords')
```

Comment:
`db_column` is not needed, since it's a `ManyToManyField` there is no column in the main table.